settings/src/main/java/org/odk/collect/settings/keys/MetaKeys.kt

```diff
@@ -10,4 +10,5 @@ object MetaKeys {
     const val EXISTING_PROJECT_IMPORTED = "existing_project_imported"
     const val LAST_LAUNCHED = "last_launched"
     const val LAST_USED_PEN_COLOR = "last_used_pen_color"
+    const val PERMISSIONS_REQUESTED = "permissions_requested"
```

Comment:
If we ask for this permission after creating a project shouldn't this be a project-level setting? Then after creating a new project, we would ask for this permission again if not granted.

Reply:
I think it makes sense to only ask once as the permissions themselves are app-level (not project-level) and I don't think we want to pester the user with multiple requests after the first (although Android will block more than two on newer OSs) if they've explicitly skipped it.